p2p/netaddress.go

```diff
@@ -281,6 +281,16 @@ func (na *NetAddress) HasID() bool {
 	return string(na.ID) != ""
 }
 
+// Endpoint converts the address to an MConnection endpoint.
+func (na *NetAddress) Endpoint() Endpoint {
+	return Endpoint{
+		Protocol: MConnProtocol,
```

Comment:
Will this be configurable somehow in the future?

Reply:
Yes. But this will require protocol changes for PEX, since we currently only exchange the IP/port numbers but not any information about which protocol to use.